sipa/utils/__init__.py

```diff
@@ -159,6 +159,25 @@ def meetingcal():
     return next_meetings
 
 
+def subscribeStatusPage(url: str, email: str) -> bool | None:
+    try:
+        response = requests.post(url, timeout=1,
+                                 headers={
+                                     "Authorization": "Token " + current_app.config['STATUS_PAGE_AUTH'],
```

Comment:
Please be consistent and use a parameter for this like in the other cases instead of accessing `current_app`